qiskit/primitives/base_estimator.py

```diff
@@ -197,28 +228,28 @@ def parameters(self) -> tuple[ParameterView, ...]:
         """
         return self._parameters
 
-    @abstractmethod
+    @deprecate_arguments({"circuit_indices": "circuits", "observable_indices": "observables"})
```

Comment:
I might be wrong, but shouldn't the deprecated arguments still be part of the signature? Otherwise I think they might become part of the `run_options` dictionary.

Reply:
I'll check it when I add the test.

Reply:
When I added the test in 6c3fe81, and the current code works well.